Description: Fix spelling errors in source tree
Author: Peter J. Mello <admin@petermello.net>
Origin: other
Forwarded: no
Last-Update: 2020-08-02

--- a/data/unitedit.ui
+++ b/data/unitedit.ui
@@ -730,7 +730,7 @@ For non-linear relations use \x for the
                       <object class="GtkBox" id="hbox1">
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
-                        <property name="tooltip_text" translatable="yes">- Decides which units the base unit is mixed with if multple options exist.
+                        <property name="tooltip_text" translatable="yes">- Decides which units the base unit is mixed with if multiple options exist.
 - The original unit will not be mixed with units with lower priority.
 - A lower value means higher priority.</property>
                         <property name="spacing">6</property>
--- a/po/es.po
+++ b/po/es.po
@@ -4473,7 +4473,7 @@ msgid "Open floating point conversion"
 msgstr "Abrir conversión de punto flotante"
 
 #: ../src/callbacks.cc:5531
-msgid "Open calender conversion"
+msgid "Open calendar conversion"
 msgstr "Abrir conversión de calendario"
 
 #: ../src/callbacks.cc:5532
--- a/po/nl.po
+++ b/po/nl.po
@@ -3296,7 +3296,7 @@ msgstr "<b>Relatie</b>"
 
 #: ../data/unitedit.ui.h:39
 msgid ""
-"- Decides which units the base unit is mixed with if multple options exist.\n"
+"- Decides which units the base unit is mixed with if multiple options exist.\n"
 "- The original unit will not be mixed with units with lower priority.\n"
 "- A lower value means higher priority."
 msgstr ""
@@ -3534,7 +3534,7 @@ msgid ""
 "(variables, functions, etc.), mode, preferences, and history of the last "
 "closed window will be saved.\n"
 "\n"
-"Do you, despite this, want to change the default bahvior and allow multiple "
+"Do you, despite this, want to change the default behavior and allow multiple "
 "simultaneous instances?"
 msgstr ""
 
@@ -4536,7 +4536,7 @@ msgstr "Geen extra conversie"
 
 #: ../src/callbacks.cc:5537
 #, fuzzy
-msgid "Open calender conversion"
+msgid "Open calendar conversion"
 msgstr "Kalenderconversie"
 
 #: ../src/callbacks.cc:5538
--- a/po/pt_BR.po
+++ b/po/pt_BR.po
@@ -4476,7 +4476,7 @@ msgid "Open floating point conversion"
 msgstr "Abrir conversão de ponto flutuante"
 
 #: ../src/callbacks.cc:5537
-msgid "Open calender conversion"
+msgid "Open calendar conversion"
 msgstr "Abrir conversão de calendário"
 
 #: ../src/callbacks.cc:5538
--- a/po/sl.po
+++ b/po/sl.po
@@ -4450,7 +4450,7 @@ msgid "Open floating point conversion"
 msgstr "Odpri orodje za pretvorbo plavajoče vejice"
 
 #: ../src/callbacks.cc:5537
-msgid "Open calender conversion"
+msgid "Open calendar conversion"
 msgstr "Odpri orodje za pretvorbo koledarjev"
 
 #: ../src/callbacks.cc:5538
--- a/po/sv.po
+++ b/po/sv.po
@@ -3258,7 +3258,7 @@ msgstr "<b>Relation</b>"
 
 #: ../data/unitedit.ui.h:39
 msgid ""
-"- Decides which units the base unit is mixed with if multple options exist.\n"
+"- Decides which units the base unit is mixed with if multiple options exist.\n"
 "- The original unit will not be mixed with units with lower priority.\n"
 "- A lower value means higher priority."
 msgstr ""
@@ -3499,7 +3499,7 @@ msgid ""
 "(variables, functions, etc.), mode, preferences, and history of the last "
 "closed window will be saved.\n"
 "\n"
-"Do you, despite this, want to change the default bahvior and allow multiple "
+"Do you, despite this, want to change the default behavior and allow multiple "
 "simultaneous instances?"
 msgstr ""
 "Enligt förinställningar tillåts enbart en instans av %s.\n"
@@ -4461,7 +4461,7 @@ msgid "Open floating point conversion"
 msgstr "Öppna flyttalsomvandling"
 
 #: ../src/callbacks.cc:5537
-msgid "Open calender conversion"
+msgid "Open calendar conversion"
 msgstr "Öppna kalenderomvandling"
 
 #: ../src/callbacks.cc:5538
--- a/src/callbacks.cc
+++ b/src/callbacks.cc
@@ -5534,7 +5534,7 @@ const gchar *shortcut_type_text(int typ
 		case SHORTCUT_TYPE_PLOT: {return _("Open plot functions/data"); break;}
 		case SHORTCUT_TYPE_NUMBER_BASES: {return _("Open convert number bases"); break;}
 		case SHORTCUT_TYPE_FLOATING_POINT: {return _("Open floating point conversion"); break;}
-		case SHORTCUT_TYPE_CALENDARS: {return _("Open calender conversion"); break;}
+		case SHORTCUT_TYPE_CALENDARS: {return _("Open calendar conversion"); break;}
 		case SHORTCUT_TYPE_PERCENTAGE_TOOL: {return _("Open percentage calculation tool"); break;}
 		case SHORTCUT_TYPE_PERIODIC_TABLE: {return _("Open periodic table"); break;}
 		case SHORTCUT_TYPE_UPDATE_EXRATES: {return _("Update exchange rates"); break;}
--- a/src/main.cc
+++ b/src/main.cc
@@ -408,7 +408,7 @@ static gint qalculate_command_line(GtkAp
 			gtk_text_buffer_set_text(GTK_TEXT_BUFFER(gtk_builder_get_object(main_builder, "expressionbuffer")), calc_arg.c_str(), -1);
 			execute_expression();
 		} else if(allow_multiple_instances < 0) {
-			GtkWidget *edialog = gtk_message_dialog_new(GTK_WINDOW(gtk_builder_get_object(main_builder, "main_window")), GTK_DIALOG_DESTROY_WITH_PARENT, GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO, _("By default, only one instance (one main window) of %s is allowed.\n\nIf multiple instances are opened simultaneously, only the definitions (variables, functions, etc.), mode, preferences, and history of the last closed window will be saved.\n\nDo you, despite this, want to change the default bahvior and allow multiple simultaneous instances?"), "Qalculate!");
+			GtkWidget *edialog = gtk_message_dialog_new(GTK_WINDOW(gtk_builder_get_object(main_builder, "main_window")), GTK_DIALOG_DESTROY_WITH_PARENT, GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO, _("By default, only one instance (one main window) of %s is allowed.\n\nIf multiple instances are opened simultaneously, only the definitions (variables, functions, etc.), mode, preferences, and history of the last closed window will be saved.\n\nDo you, despite this, want to change the default behavior and allow multiple simultaneous instances?"), "Qalculate!");
 			allow_multiple_instances = gtk_dialog_run(GTK_DIALOG(edialog)) == GTK_RESPONSE_YES;
 			save_preferences(false);
 			gtk_widget_destroy(edialog);
