Description: Update Freedesktop entry file to current standards
Author: Peter J. Mello <admin@petermello.net>
Origin: other
Forwarded: no
Last-Update: 2020-08-01

--- a/data/qalculate-gtk.desktop.in
+++ b/data/qalculate-gtk.desktop.in
@@ -1,14 +1,28 @@
 [Desktop Entry]
-Version=1.0
+Version=1.1
 _Name=Qalculate!
+Name[pt]=Qalcular!
 _GenericName=Calculator
+GenericName[pt]=Calculadora
 _Comment=Powerful and easy to use calculator
+Comment[pt]=Calculadora avançada com memória de resultados, conversões de unidades e constantes científicas
-TryExec=qalculate-gtk
+TryExec=/usr/bin/qalculate-gtk
 Exec=qalculate-gtk
 Icon=qalculate
 Terminal=false
 Type=Application
 StartupNotify=true
-StartupWMClass=qalculate-gtk
+StartupWMClass=Qalculate-gtk
-Categories=GTK;Utility;Calculator;Science;Math;
+Categories=GTK;Utility;Calculator;Science;Math;NumericalAnalysis;
-_Keywords=calculation;arithmetic;scientific;financial;
+_Keywords=math;converter;units;conversion;constants;elements;planets;programming;
+Keywords[pt]=cálculo;conversor;unidades;conversão;constantes;elementos;planetas;programação;
+DBusActivatable=false
+NoDisplay=false
+Hidden=false
+Actions=NewInstance;
+
+[Desktop Action NewInstance]
+Name=Open a new instance
+Name[pt]=Abrir uma nova sessão
+Icon=window-duplicate
+Exec=qalculate-gtk --new-instance
