Description: Update Freedesktop entry file to current standards
Author: Peter J. Mello <admin@petermello.net>
Origin: other
Forwarded: no
Last-Update: 2020-08-01

--- a/data/qalculate-gtk.appdata.xml.in
+++ b/data/qalculate-gtk.appdata.xml.in
@@ -1,23 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2003-2007, 2008, 2016-2019 Hanna Knutsson <hanna.knutsson@protonmail.com> -->
-<component type="desktop">
- <id>qalculate-gtk.desktop</id>
- <metadata_license>CC0-1.0</metadata_license>
- <project_license>GPL-2.0+</project_license>
+<!-- Copyright 2003-2008, 2016-2020,
+     Hanna Knutsson <hanna.knutsson@protonmail.com> -->
+<component type="desktop-application"
+           xmlns="https://specifications.freedesktop.org/metainfo/1.0">
+  <id>io.github.Qalculate.GTK</id>
+    <metadata_license>MIT</metadata_license>
+    <project_license>GPL-2.0-or-later</project_license>
  <_name>Qalculate! (GTK+ UI)</_name>
- <_summary>Powerful and easy to use calculator</_summary>
+    <_summary>Advanced calculator app with conversions and more</_summary>
+    <icon type="stock">io.github.Qalculate</icon>
  <description>
   <_p>
    Qalculate! is a multi-purpose cross-platform desktop calculator.
    It is simple to use but provides power and versatility normally
    reserved for complicated math packages, as well as useful tools for
    everyday needs (such as currency conversion and percent calculation).
   </_p>
   <_p>
    Features include a large library of customizable functions, unit
    calculations and conversion, physical constants, symbolic calculations
    (including integrals and equations), arbitrary precision, uncertainty
    propagation, interval arithmetic, plotting, and a user-friendly interface.
   </_p>
  </description>
  <screenshots>
--- a/data/qalculate-gtk.desktop.in
+++ b/data/qalculate-gtk.desktop.in
@@ -1,14 +1,34 @@
 [Desktop Entry]
-Version=1.0
+Version=1.1
+Type=Application
 _Name=Qalculate!
+Name[en]=Qalculate!
+Name[pt]=Qalcular!
 _GenericName=Calculator
+GenericName[en]=Multi-function calculator
+GenericName[pt]=Calculadora multifuncional
 _Comment=Powerful and easy to use calculator
+Comment[en]=Advanced calculator with currency/unit conversions, result history and scientific constants
+Comment[pt]=Calculadora avançada com conversões de dinheiro/medição, registo de resultados e constantes científicas
-TryExec=qalculate-gtk
+TryExec=/usr/bin/qalculate-gtk
 Exec=qalculate-gtk
-Icon=qalculate
+Icon=io.github.Qalculate
+Categories=GTK;Utility;Calculator;Science;Math;NumericalAnalysis;
-Terminal=false
-Type=Application
 StartupNotify=true
-StartupWMClass=qalculate-gtk
+StartupWMClass=Qalculate-gtk
-Categories=GTK;Utility;Calculator;Science;Math;
 _Keywords=calculation;arithmetic;scientific;financial;
+Keywords[en]=math;measurements;converter;elements;planets;chemistry;programming;engineering;
+Keywords[pt]=matemática;medição;conversor;elementos;planetas;química;programação;engenharia;
+Terminal=false
+NoDisplay=false
+Hidden=false
+DBusActivatable=false
+Actions=NewInstance;
+
+[Desktop Action NewInstance]
+Name=Open new instance
+Name[en]=Open another instance
+Name[es]=Abrir una nueva ventana
+Name[pt]=Abrir outra sessão
+Icon=window-duplicate
+Exec=qalculate-gtk --new-instance
