Description: Remove Encoding field in desktop file
Author: Vincent Legout <vincent@legout.info>

Index: qalculate-gtk/data/qalculate-gtk.desktop.in
===================================================================
--- qalculate-gtk.orig/data/qalculate-gtk.desktop.in
+++ qalculate-gtk/data/qalculate-gtk.desktop.in
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=Qalculate!
 _GenericName=Calculator
 _Comment=Powerful and easy to use calculator
